README.md

```diff
@@ -63,7 +63,7 @@ run <- neptune_init(project="MY_WORKSPACE/MY_PROJECT",
                     api_token="NEPTUNE_API_TOKEN")
 ```
 
-This code creates a Run in the project of your choice. This will be your gateway to log metadata to Neptune. 
+This code creates a Run in project of your choice. This will be your gateway to log metadata to Neptune. 
```

Comment:
This does not make the sentence grammatically more correct. Would propose the following:

```suggestion
This code creates a run in the specified project. This will be your gateway to logging metadata to Neptune.
```